### PR TITLE
Data provider degenscore

### PR DESCRIPTION
**Pull request by 0xmme**
added data provider for degenscore

### CODE REVIEW

**Comment by 0xmme** [comment]
@MarkuSchick  please review, thanks

**Review by 3Vis3 — COMMENTED** [review]
nlgtm

File: group-generators/helpers/data-providers/degenscore/index.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  provider: ethers.providers.JsonRpcProvider;
+
+  constructor() {
+    this.provider = new ethers.providers.JsonRpcProvider(
+      process.env.JSON_RPC_URL
+    );
+  }
+
+  public async getBeaconOwnersWithScore(_score: number) {
```
Comment (MarkuSchick) [inline group-generators/helpers/data-providers/degenscore/index.ts:23]: misses return value

File: group-generators/helpers/data-providers/degenscore/index.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    Object(enrichedData)
+      .keys()
+      .forEach((holder: string) => {
+        if (enrichedData[holder] >= _score) {
+          returnData[holder] = 1;
+        }
+      });
+  }
+
+  public async getBeaconOwnersWithScoreCount(_score: number) {
```
Comment (markus0x1) [inline group-generators/helpers/data-providers/degenscore/index.ts:45]: must return number. should just call `getBeaconOwnersWithScore` internally

**Review by markus0x1 — COMMENTED** [review]
Look good. Just one small fix

**Pull request merged**